packages/playground/src/weblets/tf_contracts_list.vue

```diff
@@ -179,7 +179,7 @@ const baseTableHeaders: VDataTableHeader = [
 const nodeTableHeaders: VDataTableHeader = [
   ...baseTableHeaders,
   { title: "Solution Type", key: "solutionType" },
-  { title: "Solution Name", key: "solutionName" },
+  { title: "Type", key: "deploymentType" },
```

Comment:
why did we remove the solution name?

Reply:
The solution type looks exactly like the contract type. Also, I thought the naming of solution type might be misleading as it contains networks & gateways.

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/40770501/472a93ed-6eb2-4f22-a5ef-ad915baa9857)